### PR TITLE
Amend Homebrew Installation Instructions

### PR DESCRIPTION
**Pull request by kakalak-lumberJack**
Included `brew install` command to add launcher to Applications Menu.

### CODE REVIEW

File: downloads.html [hunk 1]
```diff
@@ -131,7 +131,12 @@ <h2>macOS</h2>
     <p>macOS 10.10 or later is recommended.</p>
 
     <ul>
-      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version <code>brew install minetest</code></li>
+      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version: 
+	<ul style="list-style: none;">
+	  <li><code>brew install minetest</code></li>
+	  <li><code>brew linkapps</code></li>		
+	</ul>
```
Comment (SmallJoker) [inline downloads.html]: Indents from `<ul` to `</ul>` contain tabs. Please use spaces. (I hate me for saying that)

Comment (kakalak-lumberJack) [reply downloads.html]: Done.

**Comment by rubenwardy** [comment]
apparently linkapps is deprecated: https://github.com/macvim-dev/macvim/issues/450

File: downloads.html [hunk 2]
```diff
@@ -131,7 +131,12 @@ <h2>macOS</h2>
     <p>macOS 10.10 or later is recommended.</p>
 
     <ul>
-      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version <code>brew install minetest</code></li>
+      <li>Use <a href="http://brew.sh/">Homebrew</a> to get stable version: 
+        <ul style="list-style: none;">
+          <li><code>brew install minetest</code></li>
+          <li><code>brew linkapps</code></li>
```
Comment (Megaf) [inline downloads.html]: "if i want to use ~/Application i have to use

brew linkapps --local"
https://github.com/Homebrew/legacy-homebrew/issues/8699#issuecomment-62160576

I am not sure if this is the right way of doing it. Will try on my mac tomorrow. On a PC right now.

**Comment by Megaf** [comment]
@rubenwardy linkapps seems to still work, will tell for sure tomorrow.

**Comment by Megaf** [comment]
So, brew linkapps does indeed create a link for minetest in /Applications as expected. However it is deprecated and there is no replacement. How can we sort this? Perhaps for now we could merge it since it's the only semi-automagic way of adding Minetest to /Applications.

`
brew linkapps
Warning: Calling 'brew linkapps' is deprecated!
There is no replacement.
/usr/local/Homebrew/Library/Homebrew/cmd/linkapps.rb:18:in `linkapps'

Linking: /usr/local/opt/minetest/minetest.app
Linked 1 app to /Applications
`

Although in the past I remember using the command brew install minetest and it would install the .app to /Applications. I do not know what changed.

File: downloads.html (hunk 2 above)
Comment (Megaf) [reply downloads.html]: https://github.com/minetest/minetest.github.io/pull/119#issuecomment-371425277

**Comment by Megaf** [comment]
So, a normal install for an application in Brew would look like.
```
==> Verifying checksum for Cask libreoffice
==> Installing Cask libreoffice
==> Moving App 'LibreOffice.app' to '/Applications/LibreOffice.app'.
```

So we have to fix the Formula in homebrew to address this instead of changing the documentation.

**Comment by Megaf** [comment]
We need someone to make a cask instead of a formula. And host the .app in minetest.net

**Comment by alicerunsonfedora** [comment]
Theoretically, for hosting the .app file, you could just use GitHub's releases feature and just make an anchor reference to it. This is how I do it when I'm trying to reference the latest build of my DDLC mod (we regularly use Travis CI and GitHub releases).

You could probably also use some variables in your `_config.yml` and link it this way, too:
```yaml
macos_version: "0.4.17.1"
```
```html
<a href = "https://github.com/minetest/minetest/releases/download/{{ site.macos_version }}/minetest_{{ site.macos_version }}_darwinx64.zip" class = "btn btn-light">Get for macOS</a>
```
When you do it this way, when you update `macos_version`, Jekyll will automatically change the link when uploading to GitHub Pages to the latest version and point to the new release on GitHub. Granted, this is making a few assumptions:
* The release is uploaded to the releases on GitHub in the repository `minetest/minetest`.
* The release is a compressed version of the Minetest app and is named `minetest_(version)_darwinx64.zip`

**Pull request closed without merging**